plugin/priv_cc.lua

```diff
@@ -0,0 +1,8 @@
+-- if you need to set a special name for privilege, use "beerchat.name_priv_cc = spec_name" in minetest.conf
+local name_priv2cc = minetest.settings:get("beerchat.priv_cc.name")
+if name_priv2cc == '' then name_priv2cc = 'chat_creator' end
```

Comment:
I feel ``chat_creator`` should be ``beerchat_channel_creator`` or at least ``channel_creator``